### DISCUSSION TITLE
Connect Cells in Series and then simulate the series connection

### DISCUSSION THREAD

**Discussion by Azizbs1130** [pybamm-team/PyBaMM#3360] [Q&A] [answered]
Hello Everyone, 

i have been using PyBaMM for a while now but not also that advanced. I wanted to ask if there is a way to connect more cells in series and get the results of the Simulation?

**Comment by TomTranter** [comment, marked as answer]
You can use liionpack for this https://github.com/pybamm-team/liionpack
